includes/emails/class-wc-email.php

```diff
@@ -560,7 +561,7 @@ public function style_inline( $content ) {
 			if ( $this->supports_emogrifier() ) {
 				$emogrifier_class = '\\Pelago\\Emogrifier';
```

Comment:
Not directly related to this PR, but just out of curiosity in case you know, I was wondering why we are using two backslashes instead of just one. Is there a reason for that? It seems to me that just a single backslash works (`\Pelago\Emogrifier`) and it is cleaner, but I don't know if there are any problems.

Reply:
Works like that: `\Pelago\Emogrifier` is a class, while `\\Pelago\\Emogrifier` is just a string, since we need to check with `class_exists()`, we can't use just the class, because may not exists yet.

Reply:
Also since we are using single quotes: https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.single

> To specify a literal single quote, escape it with a backslash (\). **To specify a literal backslash, double it (\\). All other instances of backslash will be treated as a literal backslash:** this means that the other escape sequences you might be used to, such as \r or \n, will be output literally as specified rather than having any special meaning.

So using `\\` prevents any trouble.